mkdocs/docs/HPC/getting_started.md

```diff
@@ -0,0 +1,259 @@
+```
+
+This command returns a job identifier (*{{jobid}}*) on the HPC cluster. This is a unique identifier for the job which can be used to monitor and manage your job.
+
+!!! Warning
+ 
+    Please take note that the module commands exclusively modify environment variables. For instance, using `module swap cluster/{{othercluster}}` will instruct `qsub` to submit the job to the {{othercluster}} cluster, 
+    but our active shell session is still running on the login node.
+    
+    It is important to understand that while `module` commands affect your session environment, they do <b style="color:orange">not</b> change where the commands your are running are being executed: they will still be run on the login node you are on.
+    
+    But when submitting a job script, the commands <b style="color:orange">in</b> the job script will be run on a workernode of the cluster the job was submitted to (like `{{othercluster}}`).
+
+For detailed information about `module` commands, read the [running batch jobs](running_batch_jobs.md) chapter.
+
+### Wait for job to be executed
+
+Your job is first put into a queue before being executed, so it can take a while.
+(see [when will my job start?](running_batch_jobs.md#when-will-my-job-start) for scheduling policy)
+
+You can get an overview of the active jobs using the `qstat` command:
+<pre><code>$ qstat
+Job ID     Name             User            Time Use S Queue
+---------- ---------------- --------------- -------- - -------
+{{jobid}}     run.sh           {{userid}}        0:00:00  <b style="color:orange">Q</b> {{othercluster}}
+</code></pre> 
+
+Eventually, after entering `qstat` again you should see that your job has started running:
+<pre><code>$ qstat
+Job ID     Name             User            Time Use S Queue
+---------- ---------------- --------------- -------- - -------
+{{jobid}}     run.sh           {{userid}}        0:00:01  <b style="color:green">R</b> {{othercluster}}
+</code></pre> 
+
+If you don't see your job in the output of the `qstat` command anymore, your job has likely completed.
+
+Read [this section](running_batch_jobs.md#monitoring-and-managing-your-jobs) on how to interpret the output.
+
+### Inspect your results
+
+When your job finishes it generates 2 output files:
+
+- One for normal output messages (*stdout* output channel).
+- One for warning and error messages (*stderr* output channel).
+
+By default located in the directory where you issued `qsub`.
+
+{%- if site == 'Gent' %}
+
+!!! Info
+
+    For more information about standard input/output, see this [section](./only/gent/linux-tutorial/beyond_the_basics.md#inputoutput).
+
+{%- endif %}
+
+In our example when running <code>ls</code> in the current directory you should see 2 new files:
+ 
+- **run.sh.o{{jobid}}**: normal output messages
+- **run.sh.e{{jobid}}**: error and warning messages
+
+!!! Info
+    
+    run.sh.e{{jobid}} should be mostly empty (no error messages). If you remove the `time` command from `run.sh`, 
```

Comment:
```suggestion
    `run.sh.e{{jobid}}` should be mostly empty (no error messages). If you remove the `time` command from `run.sh`,
```

It probably makes more sense to drop the `time` command from the script, so you can simply state here that the `.e` file should be totally empty (no errors or warnings produced by the job).